spec/models/link_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Link, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user) }
+  end
+
+  context 'validations' do
```

Comment:
Use a `describe` in place of `context`. There are multiple occurrences of this.

http://betterspecs.org/